### DISCUSSION TITLE
大佬们，这是为什么

### DISCUSSION THREAD

**Discussion by floodstream** [AlistGo/alist#6897] [Q&A] [answered]
前段时间没怎么用，今天发现打不开了
[AD] Starting AList...
INFO[2024-07-29 10:00:46] reading config file: C:\Users\HHX\AppData\Roaming\ci.nn.alist\config.json
FATA[2024-07-29 10:00:46] load config error: readObjectStart: expect { or n, but found , error found in #0 byte of ...||..., bigger context ...||...
[AD] Command finished with code 1 and signal null

**Comment by MJSummer** [comment]
我也是，但是你这个提问没有可读性，感觉像是config出了问题

**Reply by floodstream** [reply]
老哥，请问你解决了吗？我看了一下config文件，是空白的，卸载重装了也不行，不知道是不是这个原因

**Reply by MJSummer** [reply, marked as answer]
把config删掉就好了 路径在roaming里面，得找一找

**Reply by floodstream** [reply]
解决了，感谢老哥
